WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -311,29 +311,31 @@ final class OrderListViewModel {
         )
     }
 
-    func feedbackBannerSurveySource() -> SurveyViewController.Source? {
+    func feedbackBannerSurveySource(onCompletion: (SurveyViewController.Source) -> Void ) {
```

Comment:
`-> Void ) ` Empty space after Void

Reply:
Removed: a939f4566362fac42294a86892bc1518783c91ea , interestingly SwiftLint didn't complain about it 🤔